src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -795,7 +828,7 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 		tabActionsContainer.classList.add('tab-actions');
 		tabContainer.appendChild(tabActionsContainer);
 
-		const tabActionRunner = new EditorCommandsContextActionRunner({ groupId: this.group.id, editorIndex: index });
+		const tabActionRunner = new EditorCommandsContextActionRunner({ groupId: this.tabsModel.id, editorIndex: this.getEditorIndex(tabIndex) });
 
```

Comment:
I think this will not work, this method needs to keep using indeces and not editors because it is called once and editors might change. The `editorIndex` is unfortunately a global thing and not a local thing here which depends on the other tab bar. I think we need a new way here to do this.

Reply:
This is why I am calling `this.getEditorIndex(tabIndex)`, this allows me the get the global index